django_bestiary/projects/templates/projects/project.html

```diff
@@ -57,7 +57,7 @@ <h1 style='display:inline'>Scope</h1>
 <!-- RIGHT COLUMN -->
 <div>
     <div class="form-group">
-      <form>
+      <form action=/projects/>
```

Comment:
I had to add this line because I got an error after importing a file. 
Once imported, the URL shows as `/projects/import`; so if you click on any option from this panel it will take you to `/projects/import?option` (which doesn't exist). With `action=/projects/`, we avoid this problem as each option will take you to `/projects/?option` as it should be.